[feature] a minimalist footer

ponsfrilus opened the pull request:
This add a footer with link to
- about page
- the version (linked to GitHub release)
- GitHub issue
- help
- source

Update the catalog page with new footer.

Closes #10

williambelle commented on src/public/css/footer.css:
```diff
@@ -0,0 +1,6 @@
+  text-align: center;
+}
+.footer-content {
+  font-size: 0.6em;
```
I think it's too small. Maybe `.875rem`? (`class="text-sm"`)

ponsfrilus replied:
Fixed

williambelle reviewed (commented):
(We can replace all the CSS with Bootstrap classes.)

ponsfrilus committed:
[feature] a minimalist footer

This add a footer with link to
- about page
- the version (linked to GitHub release)
- GitHub issue
- help
- source

Update the catalog page with new footer.

Closes #10
